hpOneView/resources/storage/storage_systems.py

```diff
@@ -184,7 +184,8 @@ def get_managed_ports(self, id_or_uri, port_id_or_uri=''):
         else:
             uri = self._client.build_uri(id_or_uri) + "/managedPorts"
 
-        return self._client.get(uri)
+        result = self._client.get(uri)
```

Comment:
You cold use the method get_collection instead, which basically is a wrapper for this line and the next line.